README.md

```diff
@@ -89,6 +89,12 @@ Install the dependencies with:
 python3 setup.py develop
 ```
 
+Install required rpm packages:
+
+```bash
+dnf install -y python-devel fedora-packager gcc krb5-devel openldap-devel gobject-introspection-devel cairo-gobject-devel
```

Comment:
Would it make sense to drop the package list to `yum-packages-devel.txt` (similar to `yum-packages.txt`)?

```suggestion
sudo dnf install $(sed 's/#.*//' yum-packages-devel.txt)
```

```
❯ cat yum-packages-devel.txt
fedora-packager
gcc
krb5-devel
openldap-devel
python-devel

# pygobject dependencies
cairo-gobject-devel
gobject-introspection-devel
```

Reply:
Put it all in a file is ok for me. Just curious about why adding sudo and removing -y?

Reply:
`sudo` is probably needed for most user but I'm not sure what's the convention for putting it into docs. It's OK to leave it out because it is clear when it is required.

I removed `-y` so that user/dev has chance to review the list of packages before they are installed. I would prefer to have the flag only in scripts.

Reply:
> I removed -y so that user/dev has chance to review the list of packages before they are installed. I would prefer to have the flag only in scripts.
Ok. Sounds reasonable to me.

Reply:
Updated